### PR TITLE
Added negation across all factTypes

### PR DESCRIPTION
**Pull request by nazrhom**
Fixes: https://github.com/resin-io/pinejs/issues/70

### CODE REVIEW

**Comment by Page-** [comment]
add to whitelist

**Comment by Page-** [comment]
add to whitelist

File: test/mocha.opts [hunk 1]
```diff
@@ -3,3 +3,4 @@
 --compilers coffee:coffee-script/register
 --bail
 --timeout 5000
+test/pilots.coffee
```
Comment (Page-) [inline test/mocha.opts]: You need to remove this

Comment (nazrhom) [reply test/mocha.opts]: good catch

**Comment by dfunckt** [comment]
Nice!

**Pull request merged**